docs: add explanation of disabled lint for Biome (#82805)
vercel/next.js: docs/01-app/03-api-reference/06-cli/next.mdx

| `-h, --help`                       | Show all available options.                                                                                                                                                                     |
| `[directory]`                      | A directory on which to build the application. If not provided, the current directory will be used.                                                                                             |
| `-d` or `--debug`                  | Enables a more verbose build output. With this flag enabled additional build output like rewrites, redirects, and headers will be shown.                                                        |
|                                    |
| `--profile`                        | Enables production [profiling for React](https://react.dev/reference/react/Profiler).                                                                                                           |

> | `--no-lint`                        | Disables linting. _Note: linting will be removed from `next build` in Next 16 for performance reasons. If you're using Next 15.5+ with Biome as a linter, linting during build will not occur._ |

Reviewer: ```suggestion
| `--no-lint`                        | Disables linting. _Note: linting will be removed from `next build` in Next 16. If you're using Next 15.5+ with a linter other than `eslint`, linting during build will not occur._ |
```
